src/App.svelte

```diff
@@ -31,8 +33,17 @@
     connectionDialogState,
   } from './script/stores/connectDialogStore';
   import { isLoading } from 'svelte-i18n';
+  import { fetchBrowserInfo } from './script/utils/api';
+  import { get } from 'svelte/store';
+
+  let isPotentiallyNonNextGenUser: boolean = false;
+  onMount(async () => {
+    if (!get(hasSeenAppVersionRedirectDialog)) {
+      const { country } = await fetchBrowserInfo();
+      const nextGenAvailableCountries = ['GB', 'JE', 'IM', 'GG'];
+      isPotentiallyNonNextGenUser = !nextGenAvailableCountries.includes(country || '');
```

Comment:
I find it strange that TypeScript complains about country being undefined, as it's perfectly valid code without the `|| ''`, nevermind.